### PR TITLE
Fixes #206: Emoji-Heatmapper app

### LINKED ISSUE #206: Emoji Heatmapper - Heatmap? Propose and implement heatmap display
There are different ways to make a heatmap. Right now the implementation seems to be to show all emojis on a map. This is not a heatmap in the original sense.

Please propose and implement an additional way to show the heatmap using emoji data in the original sense of a heatmap. A quick search on Google shows mostly examples similar to the following:
https://www.youtube.com/watch?v=9Qv7F_43dOk
https://www.youtube.com/watch?v=eepKvmknGr8
twitter-heat-map

mapsmania

r-twitter-heat-map-large570

### PR DESCRIPTION
**Pull request by kavithaenair**
### Short description
Fixes #206 
Actual emoji heatmap (works for one emoji at a time).
Demo link: https://kavithaenair.github.io/emojiHeatmapper/emoji.html
In the next iteration i will add a proper initial page (index.html) from where it redirects to emoji heatmap and emoji map. (Issue created #233 )

**Screenshots for the change**
image

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by kavithaenair** [comment]
@djmgit @singhpratyush @Achint08 @SKrPl @vibhcool Please review.

File: emojiHeatmapper/emoji-map.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
+// Geometries
+var point = new ol.geom.Point(
+    ol.proj.transform([3,50], 'EPSG:4326', 'EPSG:3857')
+    );
+
```
Comment (djmgit) [inline emojiHeatmapper/emoji-map.js]: indentation problem

**Comment by kavithaenair** [comment]
@djmgit , okay. I will try giving different color set. 👍

**Comment by kavithaenair** [comment]
Changed it @djmgit 

image

**Pull request merged**